systers_portal/common/tests/selenium/test_can_goto_common_pages.py

```diff
@@ -0,0 +1,27 @@
+from common.tests.selenium.base import SeleniumTestCase
+
+
+class TestCanGoToCommonPages(SeleniumTestCase):
+
+    def test_can_goto_home_page(self):
+        self.browser.get(self.live_server_url)
+        welcome = self.browser.find_element_by_tag_name('h1').text
+        self.assertTrue(welcome == 'Welcome to Systers Portal!')
+
+    def test_can_goto_about_page(self):
+        self.browser.get('{0}{1}'.format(self.live_server_url, '/about-us'))
```

Comment:
Same as above, you need not specify `{0}{1}` unless you are using them out of order.